nilearn/plotting/tests/test_find_cuts.py

```diff
@@ -32,6 +32,16 @@ def test_find_cut_coords():
                                # pass. x, y, z = [24.75, 3.17, 9.875]
                                rtol=6e-2)
 
+    # regression test (cf. #473)
+    # some values in image used to crash largest_connected_component()
```

Comment:
Better comment? Something like: test with no data above the activation threshold